Resources/public/templates/tabs/locations.hbt

```diff
@@ -29,7 +29,16 @@
                                         data-main-location="{{#if isMainLocation}}1{{else}}0{{/if}}">
                                 </td>
                                 <td>
-                                    <a href="{{ path "viewLocation" id=id languageCode=contentInfo.mainLanguageCode }}">{{ pathString }}</a>
+                                    <ul class="ez-breadcrumbs-list">
+                                        {{#each this.path}}
+                                        <li class="ez-breadcrumbs-item">
+                                            <a href="{{path "viewLocation" id=id languageCode=contentInfo.mainLanguageCode}}">{{contentInfo.name}}</a>
+                                        </li>
+                                        {{/each}}
+                                        <li class="ez-breadcrumbs-item">
+                                            <a href="{{path "viewLocation" id=id languageCode=contentInfo.mainLanguageCode}}">{{contentInfo.name}}</a>
```

Comment:
Point by @crevillo was not to remove the spaces inside `{{ }}`, but to **add** missing ones around `contentInfo.name`...